Out/StandaloneLinux64/bestiary.dfmod.manifest

```diff
@@ -18,27 +18,10 @@ Assets:
 - Assets/Game/Mods/Bestiary/Bestiary.dfmod.json
 - Assets/Game/Mods/Bestiary/Textures/button_gargoyle.png
 - Assets/Game/Mods/Bestiary/Textures/button_sabertooth_tiger.png
-- Assets/Game/Mods/Bestiary/Resources/entry_fire_atronach.txt
-- Assets/Game/Mods/Bestiary/Resources/entry_daedroth.txt
-- Assets/Game/Mods/Bestiary/Resources/entry_rat.txt
-- Assets/Game/Mods/Bestiary/Resources/page_monsters2.txt
-- Assets/Game/Mods/Bestiary/Resources/entry_imp.txt
-- Assets/Game/Mods/Bestiary/Resources/entry_flesh_atronach.txt
-- Assets/Game/Mods/Bestiary/Resources/entry_gargoyle.txt
-- Assets/Game/Mods/Bestiary/Resources/entry_orc_sergeant.txt
 - Assets/Game/Mods/Bestiary/Textures/base_background.png
```

Comment:
Not sure why this PR got rubber stamped immediately. I can still find mistakes, like adding these unnecessary untracked assets in the .dfmod.json.
Haven't had time to have a look at the mod message

Reply:
Why `base_background.png` is unnecessary untracked asset?

Reply:
Sorry I meant to click on the right. There's a lot of files under a folder literally named Untracked. Not meant to be used by mods, this is a DFU output when packaging mods with source

Reply:
Oh that. It's a `.manifest` file. I clean it up in no time. The files are not included. Just records about them

Reply:
> Not sure why this PR got rubber stamped immediately. I can still find mistakes, like adding these unnecessary untracked assets in the .dfmod.json. Haven't had time to have a look at the mod message

My bad, I only briefly skimmed through some of the changes and tried if the built version of the mod works. 
Thank you @KABoissonneault for looking through it and your advise!